addons/addon-base-post-deployment/packages/base-post-deployment/lib/plugins/steps-plugin.js

```diff
@@ -13,7 +13,7 @@
  *  permissions and limitations under the License.
  */
 
-const CreateRootUserService = require('../steps/create-root-user-service');
```

Comment:
is it permanently removed? if so, delete it. or add `TODO` / `FIXME` comment for later reference

Reply:
same for all commented code

Reply:
Add `TODO` comments